.github/workflows/disclosure-submitted.yml

```diff
@@ -303,6 +331,7 @@ jobs:
     needs:
       [
         contains-a-readme,
+        contains-a-valid-cvss,
```

Comment:
We can still track a disclosure submitted without a valid CVSS - remove this line.

Reply:
Removed 👍🏼